README.md

```diff
@@ -1,536 +1,121 @@
-DEDUP_TOKEN: eb6ee7d9b256590d
-== libFuzzer crashing input ==
-MS: 1 CMP- DE: "\x00C"-; base unit: 04e0ccacb50424e06e45f6184ad45895b6b8df8f
-0x6d,0x61,0x67,0x69,0x63,0x73,0x74,0x72,0x69,0x6e,0x67,0x34,0x74,0x72,0x69,0x6e,0x67,0x34,0x74,0x69,0x67,0x34,0x7b,0x0,0x0,0x43,0x34,0xa,0x0,0x0,0x0,
-magicstring4tring4tig4{\x00\x00C4\x0a\x00\x00\x00
-artifact_prefix='./'; Test unit written to crash-efea1e8fc83a15217d512e20d964040a68a968c3
-Base64: bWFnaWNzdHJpbmc0dHJpbmc0dGlnNHsAAEM0CgAAAA==
-reproducer_path='.'; Java reproducer written to Crash_efea1e8fc83a15217d512e20d964040a68a968c3.java
-```
-
-Here you can see the usual libFuzzer output in case of a crash, augmented with JVM-specific information.
-Instead of a native stack trace, the details of the uncaught Java exception that caused the crash are printed, followed by the fuzzer input that caused the exception to be thrown (if it is not too long).
-More information on what hooks and Java reproducers are can be found below.
-
-See `examples/BUILD.bazel` for the list of all possible example targets.
-
-## Usage
-
-### Creating a fuzz target
-
-Jazzer requires a JVM class containing the entry point for the fuzzer. This is commonly referred to as a "fuzz target" and
-may be as simple as the following Java example:
-
-```java
-package com.example.MyFirstFuzzTarget;
-
-public class MyFirstFuzzTarget {
-    public static void fuzzerTestOneInput(byte[] input) {
-        ...
-        // Call the function under test with arguments derived from input and
-        // throw an exception if something unwanted happens.
-        ...
-    }
-}
-```
-
-A Java fuzz target class needs to define exactly one of the following functions:
-
-* `public static void fuzzerTestOneInput(byte[] input)`: Ideal for fuzz targets that naturally work on raw byte input (e.g.
-  image parsers).
-* `public static void fuzzerTestOneInput(com.code_intelligence.api.FuzzedDataProvider data)`: A variety of types of "fuzzed
-  data" is made available via the `FuzzedDataProvider` interface (see below for more information on this interface).
-
-The fuzzer will repeatedly call this function with generated inputs. All unhandled exceptions are caught and
-reported as errors.
-
-The optional functions `public static void fuzzerInitialize()` or `public static void fuzzerInitialize(String[] args)`
-can be defined if initial setup is required. These functions will be called once before
-the first call to `fuzzerTestOneInput`.
-
-The optional function `public static void fuzzerTearDown()` will be run just before the JVM is shut down.
-
-#### Kotlin
+## Quick start
 
-An example of a Kotlin fuzz target can be found in
-[KlaxonFuzzer.kt](https://github.com/CodeIntelligenceTesting/jazzer/tree/main/examples/src/main/java/com/example/KlaxonFuzzer.kt).
+You can use Docker to try out Jazzer's Autofuzz mode, in which it automatically generates arguments to a given Java function and reports unexpected exceptions and detected security issues:
```

Comment:
I like the idea of starting the documentation with Autofuzz. A little more explanation about the shown command could be helpful, though (docker image, Maven dependency, function reference, ...).

Reply:
I added explanations.